.github/dependabot.yml

```
@@ -1,33 +1,39 @@
-        exclude-patterns:
-          - "xunit.analyzers"
-    ignore:
-      - dependency-name: "Moq"
-  - package-ecosystem: "github-actions"
-    directory: "/"
-    schedule:
-      interval: "weekly"
-      day: "sunday"
-      time: "02:00"
-      timezone: "Europe/Warsaw"
+version: 2
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)